### PR TITLE
Amend linked GH Issue label so known bugs show up

### PR DESCRIPTION
**Pull request by sadielbartholomew**
## Rationale

The documentation contains some links to the GitHub Issue Tracker for the project, & on the homepage there's one intended to go to known bugs where there is a pre-populated label of "bug". However, following that link, no bugs (open or closed) appear in the query return table.

I thought this seemed strange given there were OOM ~100 generic Issues up, so I looked at current labels & there is not one called "bug"; it turns out, the label corresponding to Issues that are considered bugs is (now) "Type: bug". So I think the current URL link is the incorrect one to that intended, & I have amended it to what seems correct.

In short, compare the:

* [old (built docs / processed) link](https://github.com/SciTools/cartopy/labels/bug);
* [proposed new (built docs / processed) link](https://github.com/SciTools/cartopy/labels/Type%3A%20bug).

## Implications

The relevant link in the documentation will take the reader to a query result page containing a listing of all the known bugs as Issues  (~150 at time of writing this up), rather than zero Issues.

In turn, there is no chance anyone following the link might mistakenly think there are no bugs registered for the project, & therefore perhaps lead to less duplicate reports!

### CODE REVIEW

**Comment by sadielbartholomew** [comment]
I have now signed the SciTools Contributor License Agreement (CLA) & I am commenting here as requested at the post-sign page.

**Comment by sadielbartholomew** [comment]
(The information given in the CI failure logs points to issues seemingly unrelated to the changes made, & the checks appear to be failing in such a way for all PRs open, so I am fairly confident they are not indicating with the proposed change, & there is nothing I can practicably do to fix them.

**Comment by dopplershift** [comment]
Yeah, CI is busted at the moment. I’m hoping to see someone update the CLA list to see that check pass before merging. (cc @SciTools/developers )

**Comment by lbdreyer** [comment]
@sadielbartholomew I can see you on the list of people that have signed the CLA (see [here](https://github.com/SciTools/scitools.org.uk/wiki/Contributors-License-Agreement-processing-details#where-can-i-see-a-list-of-all-of-those-that-have-signed-the-cla)) but we need to trigger the CLA-checker to run again so it can see you are now on the list. It would be nice to make this automatic so I will look into how this is configured, but in the meantime, a quick way to trigger it would be to close and re-open the PR a few moments. I will do this now (hope that's okay!) to see if it triggers the cla-checker to run again

**Comment by dopplershift** [comment]
Congrats on your first PR to Cartopy! Thanks for the contribution! Hope to see you around more in the future.
